src/plugin/manager.c

```diff
@@ -41,6 +41,23 @@ char* get_shm_name(const char* uid)
 	return shm_name;
 }
 
+/* remove an optional set of matching quotes (single or double),
+ * from the beginning and end of a string */
+char* unquote(char* val)
```

Comment:
The name for this function is a bit too unspecific. Also, as this function is used only inside `manager.c`, you might declare it `static`.

This function might have problems with input consisting only of a leading `"`. You might want to consider adding a case for that.

Reply:
> The function only changes an input string if its length is >= 2, and both characters (first and last) are the same type of quote, so it can already handle input with only a single leading quote (by retaining the quote in the string).

Okay, then this should work. Please find a better name for the function and this PR can get merged.